azkaban-common/src/main/java/azkaban/jobtype/JobTypeManager.java

```diff
@@ -410,6 +446,18 @@ public JobParams createJobParams(final String jobId, Props jobProps, final Logge
     }
   }
 
+  /**
+   * @param jobProps Properties for an Azkaban Job
+   * @return The {@link Optional} jobType for the Azkaban Job.
+   */
+  public static Optional<String> getJobType(Props jobProps) {
+    final String jobType = jobProps.getString("type", StringUtils.EMPTY);
+    if (StringUtils.isNotBlank(jobType)) {
```

Comment:
Can you make it a single line statement "return StringUtils.isNotBlank(jobType) ? Optional.of(jobType) : empty()" ?

Reply:
done